src/scrcpy.ts

```diff
@@ -55,6 +71,17 @@ function record() {
   start({ mode: 'record' });
 }
 
+async function mirrorWithAlwaysOnTop() {
+  const awake = (await askForStayAwake()) || null;
+  const screenOff = (await askForTurnScreenOff()) || null;
+  start({
+    mode: 'mirror',
+    alwaysOnTop: true,
+    stayAwake: awake,
+    screenOff: screenOff,
+  });
+}
+
```

Comment:
Ahh that makes sense, its just i use these combinations and its kinda tedious to go through all of the options when i can just call this command and the scrcpy window pops up

Reply:
<img width="455" alt="Screenshot 2022-10-30 at 5 02 26 PM" src="https://user-images.githubusercontent.com/18023153/198876222-bd5508fd-0994-4f11-a437-f93bdfce2912.png">
My thought here was that wouldn't it be easier for me to just run the command to get the window for me with all the options i want rather than going through the extension prompts to launch the window

Reply:
Makes sense. I wouldn't mind keeping this command as long as its name reflects all the stuff it asks you for. If one doesn't use it, it won't really bother them as VSC will likely surface the commands they frequently use anyway. Maybe like `mirror (specify x, y, z)`?

Another option could be to implement settings for the extension and allow users to customize defaults, then just use `mirror`.

Reply:
Yea, think settings would be a good idea. anyway can merge this pr. When I get time ill look into settings and proper implementations. Cheers